### PR TITLE
Adding libfreetype6-dev

### PR DESCRIPTION
**Pull request by hkim823**
For matplotlib in sandbox to work

### CODE REVIEW

File: util/install/vagrant.sh [hunk 1]
```diff
@@ -25,7 +25,7 @@ sudo apt-get upgrade -y
 ##
 ## Install system pre-requisites
 ##
-sudo apt-get install -y build-essential software-properties-common python-software-properties curl git-core libxml2-dev libxslt1-dev python-pip python-apt python-dev
+sudo apt-get install -y build-essential software-properties-common python-software-properties curl git-core libxml2-dev libxslt1-dev libfreetype6-dev python-pip python-apt python-dev
```
Comment (feanil) [inline util/install/vagrant.sh:28]: Why do this at all if ansible is going to get run?

Comment (hkim823) [inline util/install/vagrant.sh:28]: I ran into a scenario that ansible wasn't picking up the requirement building from a clean ubuntu box, just to be sure I thought I would add this in here.

Comment (feanil) [inline util/install/vagrant.sh:28]: @hkim823  I would opt for removing this line entirely and fixing the issue that was causing ansible to not install all system packages correctly.  Is this easily re-producible?

Comment (hkim823) [inline util/install/vagrant.sh:28]: Unsure how reproducible it is and will require longer term testing to see how ansible installs debian pacakges based on roles.

**Comment by singingwolfboy** [comment]
I'm reproducing this failure: https://gist.github.com/singingwolfboy/390df7e9da3b6f51f6d1 and it sounds like @mattdrayer is as well.

**Comment by benpatterson** [comment]
CC @clytwynec who was troubleshooting this as well.

**Comment by hkim823** [comment]
Closing this PR, this should be handled by ansible, not by relying on manual pre install intervention

**Pull request closed without merging**